### ISSUE TITLE
Provide some specific guidance on figure quality, axis labels, etc.

### ISSUE THREAD

**Issue by mrshirts** [livecomsjournal/livecomsjournal.github.io#81]
PLoS one has quite significantly detailed information on submissions:

http://journals.plos.org/plosone/s/submission-guidelines#loc-figures-and-tables
Look at, for example, their figure guidelines:
http://journals.plos.org/plosone/s/figures

and tables guidelines
http://journals.plos.org/plosone/s/tables

I'm guessing that they have guidelines this detailed because they discovered it was necessary.  So I wonder to what extent we should try to develop something like this level of detail (we have an example of what might be useful to ask for in the PLoS guidelines!)

**Comment by davidlmobley** [comment]
I think we're going to need less extensive figure guidelines than a "normal" journal because we are not typesetting or otherwise working with their articles, so we don't have to worry as much about it. Possibly ours need to be beefed up a bit, but I would not expect them to need THIS much beefing up. My take would be that we would probably want something pretty minimal, because I'll basically be happy with their figures/tables as long as they:
1. Look good in the PDF of the article, even if zoomed
2. Look good if the article is printed
3. Don't result in the manuscript being too huge a file
4. Are stylistically relatively consistent with the paper (e.g. font sizes/styles are not dramatically different)

This means that a pretty wide range of formats, etc. will be possible, and I would not want to be as detailed/specific as PLos. In my view it's more about making sure the OUTCOME is good.

**Comment by mrshirts** [comment]
Conversely, if there is no copy editing, more instructions could save time - if it doesn't look good, we point to the instructions, instead of having to explain why it doesn't look good.

**Comment by jchodera** [comment]
Related to copyediting: Overleaf has some suggestions on tools you can use with it to help check/fix grammatical and spelling errors:
https://www.overleaf.com/help/334-can-i-use-overleaf-with-language-checking-tools-such-as-grammarly

**Comment by davidlmobley** [comment]
I thought you were specifically talking about figures and tables. I'm fine with being pretty specific on grammar/style.

**Comment by dmzuckerman** [comment]
I would propose guidelines (or at least suggestions) on figure captions - namely, that they be as self-contained as reasonably possible.  A reader should be able to look at a figure and caption and essentially understand it without going to text.  If a technical quantity is being plotted it can be explained in a few words and linked out to Eq xx.  Ideally captions will have a descriptive title and not just immediately give technical details about what is plotted.

I would also like to suggest using words on axis labels: "Fractional error \epsilon" instead of just "\epsilon", "Time t (s)" instead of "t(s)".

**Comment by davidlmobley** [comment]
Totally agree with Dan here. :)

**Comment by davidlmobley** [comment]
I think we also discussed this some in a call. I think action items are:
- [ ] Provide some recommendations regarding construction of figures/figure formats which will lead to them usually looking good, recommend font (OpenSans!) and size for axis labels, etc.
- [ ] Make clear the largest issue with figures is quality in the final PDF, e.g. when zoomed, etc., as noted [here](https://github.com/livecomsjournal/livecomsjournal.github.io/issues/81#issuecomment-339662760)
- [ ] Note that size of the PDF is also an issue; don't want huge PDFs
- [ ] Make recommendation on axis labels (words and symbols)
- [ ] Figure captions should be self-contained 
- [ ] Skim PLOS guidance on tables and see if there's anything we should especially be requesting for tables here

**Comment by davidlmobley** [comment]
I'm going to submit a PR to address these momentarily. I looked over the PLoS guidance on figures and tables and feel we do not need that level of detail here, in large part because we are not typesetting the documents. Our main concern is the quality of the PDF, so we don't need to get nearly as specific as PLoS which sets very specific requirements in order to ensure their life is straightforward.

My PR will include some recommendations on how to get high quality figures/things to watch out for, but I think that's all we need to do.

PLoS also has very specific guidance as to how/how not to submit tables, but again, most of this seems to do with making their typesetting life easier (e.g. not using spaces to get a new line, etc.) and won't apply to us.

My perspective is that we mostly just want to make authors' lives easy for them by keeping requirements minimal/product-oriented (assuming they know what they are doing) by only paying attention to how the PDF looks, and providing ADVICE as needed on how to get to a good final product.

In other words, I suggest that if we find authors are struggling with how to end up with high quality figures/tables, then we provide some separate help documentation giving advice on these topics, and keep the requirements simple.

**Comment by davidlmobley** [comment]
Resolved by #109
